webhook-integrations/papertrail/notes.md

```diff
@@ -0,0 +1,21 @@
+
+Papertrail
+    Papertrail only sends logs to its integrated tools.
+    Papertrail log extract works for nearly every device or software.
+    Logs are sent via link that is generated by the integrated tools.
+    I've used a tool named Slack, created an account (team),
+    Searched for papertrail and generated a link posting the logs
+    in #general(group), Made an account in papertrailapp.com
+    and configured it using my pc's logs and set an alert for 
+    the link that was generated by slack, And therefore it worked.
+    This is the way papertrail works.
+    
+Creating a slack account in fig Task_4_5.
```

Comment:
Can you convert the Task_<foo> to markdown. E.g. ![Create Account](Task_4_5.png)?